src/authentication/styles.styl

```diff
@@ -0,0 +1,109 @@
+@require 'cozy-ui'
```

Comment:
This style is a subset of the `onboarding.styl` file, with only the parts useful for the auth screen. I'm planning on rewriting a large part of it later.